physics/PBL/SATMEDMF/satmedmfvdifq.meta

```diff
@@ -241,12 +241,12 @@
   type = real
   kind = kind_phys
   intent = in
-[icplocn2atm]
-  standard_name = control_for_air_sea_flux_computation_over_water
+[use_oceanuv]
+  standard_name = flag_for_control_of_air_sea_flux_computation_over_water
```

Comment:
Can you change this standard name to `do_air_sea_flux_computation_over_water`? We're trying to switch all logicals over to `do_X` so that it is more obvious by the name what happens when it is true.